assets/embedded-files/tekton/stage-pipeline.yaml

```diff
@@ -57,6 +67,29 @@ spec:
     workspaces:
     - name: source
       workspace: source
+    - name: cache
+      workspace: cache
+
+---
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: cleanup
+  namespace: tekton-staging
+spec:
+  workspaces:
+  - name: source
+  steps:
+  - name: cleanup
+    image: lachlanevenson/k8s-kubectl
```

Comment:
I'm just re-using what the other task uses. Anything with `rm` available would do.